tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2466,87 +2466,142 @@ TEST_P(OpConverter_FP32_Test, ConvertShape) {
   }
 }
 
+struct MatMulTestParams {
+  std::vector<int> shape_a;
+  std::vector<int> values_a;
+  bool transpose_a;
+  std::vector<int> shape_b;
+  std::vector<int> values_b;
+  bool transpose_b;
+  std::vector<int> expected_shape;
+  std::vector<int> expected_output;
+};
+
 // Helper function for testing MatMul and BatchMatMul
 // get_matmul corresponds to the function used to generate the node. It should
 // accept (DataType, transpose_a, transpose_b) as parameters.
```

Comment:
s/corresponds to the function used/is a function/
s/should accept/accepts/
Add a period after "BatchMatMul".
Please also make sure not partial lines besides the last line in the comment block.